tests/suites/0_stateless/02_function/02_0001_function_type_of.sql

```diff
@@ -31,5 +31,5 @@ select 'UInt64 OP UInt64', typeof(65536 * 65536 + 65536 * 65536), typeof(65536 *
 
 select '=== TEST_datetimes';
 
-select typeof(now()) = 'DateTime_0';
+select typeof(now()) = 'DateTime(0)';
```

Comment:
if it's zero, better to display `DateTime` ?

Reply:
sounds reasonable

Reply:
resolved